build_tools/github_actions/configure_ci.py

```diff
@@ -204,19 +204,10 @@ def get_runner_env(trailers: Mapping[str, str]) -> str:
   return runner_env
 
 
-def get_ci_stage(event_name):
-  if event_name == PULL_REQUEST_EVENT_NAME:
-    return "presubmit"
-  elif event_name == PUSH_EVENT_NAME:
-    return "postsubmit"
-  elif event_name == SCHEDULE_EVENT_NAME:
-    return "postsubmit"
-  elif event_name == WORKFLOW_DISPATCH_EVENT_NAME:
-    return "unknown"
-  raise ValueError(f"Unrecognized event name '{event_name}'")
-
-
-def get_benchmark_presets(ci_stage: str, trailers: Mapping[str, str]) -> str:
+def run_on_fork(event_name):
+  return event_name == PULL_REQUEST_EVENT_NAME
```

Comment:
This helper function confuses me. PRs don't have to originate from forks.

Reply:
But they *can* and usually do. I guess we could have PRs from the main repo also write caches and such.